src/torrent.cpp

```diff
@@ -3114,14 +3114,16 @@ namespace {
 		// so that each one should get at least one announce
 		std::vector<announce_state> listen_socket_states;
 
+		bool announce_to_all_tiers = settings().get_bool(settings_pack::announce_to_all_tiers);
+		bool announce_to_all_trackers = settings().get_bool(settings_pack::announce_to_all_trackers);
```

Comment:
Not sure about this repo policy on `const` but I would suggest adding them anyway:
```suggestion
		bool const announce_to_all_tiers = settings().get_bool(settings_pack::announce_to_all_tiers);
		bool const announce_to_all_trackers = settings().get_bool(settings_pack::announce_to_all_trackers);
```

Reply:
Whoops, I'm used to rust, where things are const by default.